lib/rubocop/cop/capybara/mixin/css_selector.rb

```diff
@@ -83,6 +85,49 @@ def multiple_selectors?(selector)
           normalize = selector.gsub(/(\\[>,+~]|\(.*\))/, '')
           normalize.match?(/[ >,+~]/)
         end
+
+        # @param selector [String]
+        # @return [String]
+        # @example
+        #   css_escape('some-id') # => some-id
+        #   css_escape('some-id.with-priod') # => some-id\.with-priod
+        # @reference
+        #   https://github.com/mathiasbynens/CSS.escape/blob/master/css.escape.js
+        def css_escape(selector) # rubocop:disable Metrics/AbcSize,Metrics/CyclomaticComplexity,Metrics/MethodLength,Metrics/PerceivedComplexity
```

Comment:
How do you feel to only add handling for the period in this PR, and leaving the rest for later?